routes/setup.js

```diff
@@ -50,50 +52,16 @@ router.get("/id", async function (req, res) {
   }
 });
 
-router.get("/featured", async function (req, res) {
+router.get("/search", async function (req, res) {
   res.setHeader("Content-Type", "application/json");
   try {
     var userId = await routeUtils.verifyLoggedIn(req, true);
```

Comment:
don't delete the /featured function, we are still using it. 

/v/ removed the functionality